extensions/BugModal/web/bug_modal.css

```diff
@@ -781,7 +820,7 @@ body.platform-Win32 .comment-text, body.platform-Win64 .comment-text {
     clear: both;
     width: 100%;
     box-sizing: border-box !important;
-    margin: 0 0 1em;
+    margin: 0 0 0.5em;
```

Comment:
@purelogiq is this change from my merge?

Reply:
I don't think so, I remember intentionally changing that because of there being more default spacing with <p> tags vs <pre> tags. Thinking of reverting this?

Reply:
Nah, just checking if I screwed up the merge